--- control.c.orig	Thu Sep 24 06:02:47 1998
+++ control.c	Wed Jan 24 15:15:29 2001
@@ -933,12 +933,12 @@
 	}
 #ifdef SANITY
 	if (t->sanity) {
-		if (PZBITS(h->ver)) {
+/*		if (PZBITS(h->ver)) {
 			if (DEBUG) {
 				log(LOG_DEBUG,"%s: Reserved bits set -- must toss\n",__FUNCTION__);
 			}
 			return -EINVAL;
-		}
+		}  */
 		if (PTBIT(h->ver)) {
 			if (DEBUG) {
 				log(LOG_DEBUG,"%s Control bit set\n",__FUNCTION__);
